### PR TITLE
fix render issue when switching to text mode

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Change to studioApp fixes https://codeorg.axosoft.com/viewitem?id=151&type=features&force_use_number=true

### CODE REVIEW

File: apps/src/StudioApp.js [hunk 1]
```diff
@@ -2194,6 +2194,12 @@ StudioApp.prototype.handleEditCode_ = function (config) {
     try {
       // Don't pass CRLF pairs to droplet until they fix CR handling:
       this.editor.setValue(startBlocks.replace(/\r\n/g, '\n'));
+      // When adding content via setValue, the aceEditor cursor gets set to be
+      // at the end of the file. For mysterious reasons we've been unable to
+      // understand, we end up with some pretty funky render issues if the first
+      // time we switch to text mode the cursor is out of view beyond the bottom
+      // of the editor. Navigate to the start so that this doesn't happen.
+      this.editor.aceEditor.navigateFileStart();
```
Comment (bcjordan) [inline apps/src/StudioApp.js]: LGTM :+1:

**Comment by bcjordan** [comment]
LGTM :+1:

**Comment by Bjvanminnen** [comment]
Eyes failure is just because cursor is in a different spot.

**Comment by bcjordan** [comment]
Re: eyes failure, you may even next time be able to go in and accept the new baseline via Circle -> Cucumber -> Applitools link, re-run the Circle build (should be green), then merge into staging. (Email coming this weekend with those instructions, still just verifying in a couple of branches the workflow works OK)

**Pull request merged**